frontend/frontend/src/components/ConfigBar/components/PresetButtons.vue

```diff
@@ -10,6 +10,11 @@
           <template #button-content>
             My Presets <b-icon icon="justify" font-scale="1.5" style="margin-left: 4em"></b-icon>
           </template>
+          <div v-for="preset in userPresetsUpdate" :key="preset[1]">
+            <b-dropdown-item>
+              {{ preset[0] }}, {{ preset[1] }}
```

Comment:
```suggestion
              {{ `{preset[0] }, {preset[1]}` }}
```

Not suggesting this just showing you another way you can do it :)